types/api-schema.ts

```diff
@@ -0,0 +1,7 @@
+export type TUser = {
```

Comment:
Does `TUser` make more sense here than just `User`? Does the `T` just stand for "type" or is it something else?

Reply:
Yeah, it just stands for Type, it was something I started doing in an Enterprise repo where people used `type` and `interface` interchangeably EVERYWHERE... I can just remove the T from the types later if preffered.